nautobot/ipam/tests/test_migrations.py

```diff
@@ -421,7 +421,8 @@ def test_aggregate_to_prefix_migration_object_changes(self):
         "mysql does not support rollbacks",
     )
     def test_aggregate_to_prefix_migration_custom_fields(self):
-        self.assertEqual(self.custom_field.objects.exclude(name="example_plugin_auto_custom_field").count(), 3)
+        # This change is necessary because name attribute is not specified now in example_plugin's signal.py
+        self.assertEqual(self.custom_field.objects.exclude(name="").count(), 3)
```

Comment:
Seems weird to me - `name` is gone as a field right? Should we instead look for specific `label` values instead of excluding empty `name`s?

Reply:
The `extras` migration that deletes the `name` field has not been run yet after `(dcim, 0030)` migration is executed in this test. So CustomField still has the `name` attribute